wls-wahlvorstand-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorstandservice/service/wahlvorstand/WahlvorstandService.java

```diff
@@ -0,0 +1,196 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorstandservice.service.wahlvorstand;
+
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.domain.wahlvorstand.Wahlvorstand;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.domain.wahlvorstand.WahlvorstandRepository;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.domain.wahlvorstand.Wahlvorstandsmitglied;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.wahlvorstandservice.security.AuthenticationHandler;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.parameters.P;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class WahlvorstandService {
```

Comment:
In der Klasse wird stark gemischt zwischen final (val) und non-final, vermutlich je nachdem was angepasst oder direkt übernommen wurde. Bitte auf `val` vereinheitlichen

Reply:
siehe commit [0624bb3](https://github.com/it-at-m/Wahllokalsystem/pull/533/commits/0624bb3a9e7e7b89c8c3be0d701667309c6ffa3c)